test/invariant/actors/Proposer.t.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import { BaseActor, IERC20, IERC20Metadata, AngleGovernor, TestStorage } from "./BaseActor.t.sol";
+import { console } from "forge-std/console.sol";
+import { IGovernor } from "oz/governance/IGovernor.sol";
+import { ProposalStore, Proposal } from "../stores/ProposalStore.sol";
+import { IERC5805 } from "oz/interfaces/IERC5805.sol";
+import { TimestampStore } from "../stores/TimestampStore.sol";
+
+contract Proposer is BaseActor {
+    AngleGovernor internal _angleGovernor;
+    ProposalStore public proposalStore;
+    IERC5805 public veANGLEDelegation;
+    TimestampStore public timestampStore;
+
+    constructor(
+        AngleGovernor angleGovernor,
+        IERC20 _agToken,
+        uint256 nbrVoter,
+        ProposalStore _proposalStore,
+        IERC5805 _veANGLEDelegation,
+        TimestampStore _timestampStore
+    ) BaseActor(nbrVoter, "Proposer", _agToken) {
+        _angleGovernor = angleGovernor;
+        proposalStore = _proposalStore;
+        veANGLEDelegation = _veANGLEDelegation;
+        timestampStore = _timestampStore;
+    }
+
+    function propose(uint256 value) public useActor(1) {
+        address[] memory targets = new address[](1);
+        bytes[] memory datas = new bytes[](1);
+        uint256[] memory values = new uint256[](1);
+        string memory description = "Test Proposal";
+        targets[0] = address(_angleGovernor);
```

Comment:
Why not randomize the content?

Reply:
We cannot really have fully randomize data as arbitrary execution of calldata can lead to a unexpected revert. The only params we can change is values and description so let me know what you think

Reply:
Indeed if we randomize the address with a null calldata like right now it shouldn't revert no?